### PR TITLE
Add the Sent Message to the Cache

### PR DESCRIPTION
**Pull request by goksunonal**
No need to check same sent message from relay.

### CODE REVIEW

File: src/network.c [hunk 1]
```diff
@@ -227,6 +227,7 @@ int network_send_msg(struct network *net, struct network_msg *nmsg)
 	nonce.type = NONCE_NETWORK;
 	memcpy(&nonce.ctl_ttl, (void *)nmsg + 1, 6);
 	nonce.iv_index = cpu_to_be32(net->iv_index);
+	err1 = network_cache_add(net, nmsg);
```
Comment (loicpoulain) [inline src/network.c]: It looks good, but you don't need to introduce a new err1 variable here, neither check returned error.

**Review by loicpoulain — CHANGES_REQUESTED** [review]
Idea is good, but change requested, cf inline comment.

File: src/network.c (hunk 1 above)
Comment (goksunonal) [reply src/network.c]: Updated...

**Comment by loicpoulain** [comment]
Could you please propose only one commit for this change.

**Pull request merged**